smaps/src/main/java/com/google/smaps/Histogram.java

```diff
@@ -35,19 +35,31 @@
   static long lowerBound;
   // Used to populate the upper bound text box and slider max.
   static long upperBound;
+  // Stores the minimum region size from entire address space.
+  static long minBound;
+  // Stores the maximum region size from entire address space.
+  static long maxBound;
```

Comment:
Can we make these final? I don't think we can make them constants since we don't know them right away/in the constructor, but making them final should help prevent them from being changed.

Reply:
I tested making them final, but then they are unable to be reassigned from zero as the actual min/max on lines 162 and 163 in the setMinMax function.

Reply:
Ah that's right, same issue as constants. Okay we will just have to leave them as is then.